component/gitlab-ci.jsonnet

```
@@ -0,0 +1,103 @@
+local kap = import 'lib/kapitan.libjsonnet';
+local inv = kap.inventory();
+local params = inv.parameters.openshift4_terraform;
+
+local cloud_specific_variables = {
+  cloudscale: {
+    default: {
+      CLOUDSCALE_TOKEN: '${CLOUDSCALE_TOKEN_RO}',
+    },
+    apply: {
+      CLOUDSCALE_TOKEN: '${CLOUDSCALE_TOKEN_RW}',
+    },
+  },
+  exoscale: {
+    default: {
+      EXOSCALE_API_KEY: '${EXOSCALE_API_KEY_RO}',
+      EXOSCALE_API_SECRET: '${EXOSCALE_API_SECRET_RO}',
+    },
+    apply: {
+      EXOSCALE_API_KEY: '${EXOSCALE_API_KEY_RW}',
+      EXOSCALE_API_SECRET: '${EXOSCALE_API_SECRET_RW}',
+    },
+  },
+};
```

I'd add the Exoscale key/secret for Ursula here, and update the Exoscale how-to accordingly:

```suggestion
local ursula_vars_exoscale = {
  TF_VAR_lb_exoscale_api_key: '${EXOSCALE_URSULA_KEY}',
  TF_VAR_lb_exoscale_api_secret: '${EXOSCALE_URSULA_SECRET}',
};
local cloud_specific_variables = {
  cloudscale: {
    default: {
      CLOUDSCALE_TOKEN: '${CLOUDSCALE_TOKEN_RO}',
    },
    apply: {
      CLOUDSCALE_TOKEN: '${CLOUDSCALE_TOKEN_RW}',
    },
  },
  exoscale: {
    default: {
      EXOSCALE_API_KEY: '${EXOSCALE_API_KEY_RO}',
      EXOSCALE_API_SECRET: '${EXOSCALE_API_SECRET_RO}',
    } + ursula_vars_exoscale,
    apply: {
      EXOSCALE_API_KEY: '${EXOSCALE_API_KEY_RW}',
      EXOSCALE_API_SECRET: '${EXOSCALE_API_SECRET_RW}',
    } + ursula_vars_exoscale,
  },
};
```